src/containers/form/FormContainer.js

```diff
@@ -341,8 +349,11 @@ class Form extends React.Component {
       const value = formInputs[propertyType.type.name];
       let formattedValue;
       formattedValue = Array.isArray(value) ? value : [value];
-      formattedValue = (formattedValue.length > 0 && (formattedValue[0] === '' || formattedValue[0] === null))
-        ? [] : formattedValue;
+      formattedValue = (
```

Comment:
Please use if-else statements as I think it'll be a little easier to grok what's happening here.

Reply:
Updated